rust/sealevel/programs/mailbox/src/accounts.rs

```diff
@@ -100,6 +100,8 @@ pub struct Outbox {
     pub owner: Option<Pubkey>,
     /// The merkle tree of dispatched messages.
     pub tree: MerkleTree,
+    /// The protocol fee configuration.
```

Comment:
I imagine we may want a max protocol fee that's set at init time, wdyt? I think it's a best practice for stuff like this

https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/solidity/contracts/hooks/ProtocolFee.sol#L38